server/src/test/java/org/elasticsearch/ExceptionSerializationTests.java

```diff
@@ -886,7 +886,7 @@ public void testShardLockObtainFailedException() throws IOException {
 
     public void testSnapshotInProgressException() throws IOException {
         SnapshotInProgressException orig = new SnapshotInProgressException("boom");
-        Version version = VersionUtils.randomVersionBetween(random(), Version.V_6_7_0, Version.CURRENT);
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_7_0_0, Version.CURRENT);
```

Comment:
no need to do here, but we could probably replace this so that we do not need to worry about it in the big 7.x refactor.

to something like

```
Version version = Versionutils.randomVersion()
```

Reply:
I think it should be `VersionUtils.randomIndexCompatibleVersion(random())`

Reply:
👍 I'll make this change in the current PR.